### PR TITLE
Alessio dynamic pricing plan

### PR DESCRIPTION
**Pull request by cpalessio96**
### All Submissions:

*   [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
*   [x] Does your code has proper inline documentation? 

### Changes proposed in this Pull Request:
I have removed all the information regarding the plans, now all the information is dynamic and is retrieved upon activation and updating of the settings wubtitle page

Closes # 155.

### CODE REVIEW

**Comment by GiuseppeIuculano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- includes/Core/Cron.php  2
- includes/Core/Activation.php  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5788292&bid=18948541)

File: includes/Core/Activation.php [hunk 1]
```diff
@@ -48,8 +48,25 @@ public function wubtitle_activation_license_key() {
...
+					// phpcs:disable 
+					// warning camel case
+					'totalJobs'    => $plan->totalJobs,
+					'totalSeconds' => $plan->totalSeconds,
+					// phpcs:enable
+					'price'        => $plan->price,
+					'dot_list'     => $plan->dotlist,
+					'icon'         => $plan->icon,
+				);
+			}
```
Comment (HyprCoder) [inline includes/Core/Activation.php:68]: Se plans è un array qui puoi semplicemente usare la funzione array_map

**Comment by GiuseppeIuculano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- includes/Core/Cron.php  2
- includes/Core/Activation.php  1
         

Clones removed
==============
+ assets/payment/settings_script.js  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5788292&bid=18948541)

**Comment by GiuseppeIuculano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- includes/Core/Cron.php  2
         

Clones removed
==============
+ assets/payment/settings_script.js  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5788292&bid=18948541)

**Pull request merged**